### ISSUE TITLE
what is " Change Token "

### ISSUE THREAD

**Issue by ITZDude1** [Moonboy05/BlooketHacks#12]
IM really confused about change token cuz i tried and i dont think it did anything

**Comment by hankypoo7** [comment]
yeah theese dont work, go to this respository and it actually works.
https://github.com/Hankypoo7/Blooket-hacks-Hankypoo7-

**Comment by PennTheGreat** [comment]
![bitmoji](https://sdk.bitmoji.com/render/panel/b6d0de0a-0762-418e-9db2-b84c3ef51251-deb8ae20-bfe6-4bfa-9a0a-09b72ccaeec0-v1.png?transparent=1&palette=1&width=246)

Like, that's LITERALLY advertising. (I don't give a ****)

**Comment by R3nH4s1ssu3s** [comment]
I'm not sure if it's necessary to advertise, but if you want, you can go to my repository :)
